src/gluonts/ext/r_forecast/_predictor.py

```diff
@@ -291,54 +242,23 @@ def _forecast_dict_to_obj(
     def predict(
         self,
         dataset: Dataset,
-        num_samples: int = 100,
-        intervals: Optional[List] = None,
-        save_info: bool = False,
         **kwargs,
     ) -> Iterator[Forecast]:
-        """
-
-        Parameters
-        ----------
-        dataset
-            Dataset of all time series.
-        num_samples
-            Number of samples to store in sample based forecast.
-        intervals
-            Prediction intervals for the quantile based forecast.
-        save_info
-            Should console output from R methods be saved?
-        kwargs
-
-        Returns
-        -------
-        Iterator[Forecast]
-            Iterator over gluonts `Forecast` object.
-
-        """
         self._warning_message()
 
         for data in dataset:
             data = self._preprocess_data(data=data)
-            params = self._override_params(
-                params=self.params.copy(),
-                num_samples=num_samples,
-                intervals=intervals,
```

Comment:
I think the reason for using separate set of `params` for each time series instead of using `self.params` is being able to run this in parallel. Not sure if it is needed though.

Reply:
That should not be an issue in case the process is forked, right @jaheba? Most importantly, if the model does not modify `self.params` on a per-series basis, then it's essentially a read-only dictionary and should be fine.

But let me verify these